genfragments/ThirteenTeV/ADD/ADDmonoPhoton_MD_1000_d_3_TuneCP5_13TeV_pythia8_cfi.py

```diff
@@ -0,0 +1,42 @@
+
+import FWCore.ParameterSet.Config as cms
+
+
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
could you please change it to concurrent mode one

Reply:
Done